addon/components/relationship-select.hbs

```diff
@@ -0,0 +1,25 @@
+<input
+  class="hidden-input"
+  value={{or (or @selected.id @selected.length) ""}}
```

Comment:
Why would that input have a value of `@selected.length`. Also, if it's hidden just add the `hidden` attribute

Reply:
The select can be a `power-select-multiple`. The hidden is so we can use html5 validation for the input . The hidden attribute didn't work for this as the input was not validated.

Reply:
Ah but i didnt know about `type="hidden"`. will try this thanks

Reply:
Ah so this input is to verify it has a value?

Reply:
Yes

Reply:
Since ember-power-select does not render an actual input element, this is the workaround so we dont have to do js validation.